### PR TITLE
[kie-roadmap#93] Build Maven repository zip

### PR DESCRIPTION
**Pull request by baldimir**
- Adds a Maven module that uses BAMOE bom to download all artifacts, and zips only the required ones into a Maven repository zip file. 

Signed-off-by: Author Name <tibor.zimanyi@ibm.com>

### CODE REVIEW

File: bamoe-maven-repository-zip/pom.xml [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-addons-quarkus-knative-eventing-deployment</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-addons-quarkus-knative-eventing-deployment</artifactId>
+      <classifier>sources</classifier>
+    </dependency>
+
+<!--    <dependency>-->
```
Comment (gitgabrio) [inline bamoe-maven-repository-zip/pom.xml:140]: @baldimir 
Is this a "remainder" or could be removed ?

File: bamoe-maven-repository-zip/pom.xml [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-quarkus-extension-spi</artifactId>
+      <classifier>sources</classifier>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-quarkus-extension-spi</artifactId>
+    </dependency>
+
+    <!-- Quarkus DevUI addon -->
```
Comment (gitgabrio) [inline bamoe-maven-repository-zip/pom.xml:742]: same comment as above

**Review by gitgabrio — APPROVED** [review]
thanks @baldimir 
Beside a couple of comments, LGTM

File: bamoe-maven-repository-zip/pom.xml (hunk 1 above)
Comment (baldimir) [reply bamoe-maven-repository-zip/pom.xml:140]: Thanks for pointing this. Yes, I forgot to uncomment.

File: bamoe-maven-repository-zip/pom.xml (hunk 2 above)
Comment (baldimir) [reply bamoe-maven-repository-zip/pom.xml:742]: Fixed.

**Comment by baldimir** [comment]
Converting to draft as I need to exclude some files from the zip.

**Pull request merged**